EasyVCR.Tests/EasyVCR.Tests.csproj

```diff
@@ -20,5 +20,6 @@
         <PackageReference Include="MSTest.TestAdapter" Version="2.2.10" />
         <PackageReference Include="MSTest.TestFramework" Version="2.2.10" />
         <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+        <PackageReference Include="RestSharp" Version="108.0.2" />
```

Comment:
Question: This will only add this dep to the test suite right? Not EasyVCR itself?

Reply:
Correct